{"level":"error","message":"Error conneting to the DB: MongooseServerSelectionError: Server selection timed out after 5000 ms"}
{"level":"error","message":"Error at getting new item: CastError: Cast to ObjectId failed for value \"0\" (type string) at path \"_id\" for model \"carts\""}
{"level":"error","message":"Error at getting new item: CastError: Cast to ObjectId failed for value \"1\" (type string) at path \"_id\" for model \"carts\""}
{"level":"error","message":"Error at getting new item: CastError: Cast to ObjectId failed for value \"undefined\" (type string) at path \"_id\" for model \"carts\""}
{"level":"error","message":"Error at getting new item: CastError: Cast to ObjectId failed for value \"undefined\" (type string) at path \"_id\" for model \"carts\""}
{"level":"error","message":"Error at getting new item: CastError: Cast to ObjectId failed for value \"0\" (type string) at path \"_id\" for model \"carts\""}
{"level":"error","message":"Error at getting new item: CastError: Cast to ObjectId failed for value \"1\" (type string) at path \"_id\" for model \"carts\""}
{"level":"error","message":"Error at getting new item: CastError: Cast to ObjectId failed for value \"1\" (type string) at path \"_id\" for model \"carts\""}
{"level":"error","message":"Error at getting new item: CastError: Cast to ObjectId failed for value \"0\" (type string) at path \"_id\" for model \"carts\""}
{"level":"error","message":"Error at getting new item: CastError: Cast to ObjectId failed for value \"[object Object]\" (type string) at path \"_id\" for model \"carts\""}
{"level":"error","message":"Error conneting to the DB: MongooseServerSelectionError: Could not connect to any servers in your MongoDB Atlas cluster. One common reason is that you're trying to access the database from an IP that isn't whitelisted. Make sure your current IP address is on your Atlas cluster's IP whitelist: https://docs.atlas.mongodb.com/security-whitelist/"}
{"level":"error","message":"Error conneting to the DB: MongooseServerSelectionError: Could not connect to any servers in your MongoDB Atlas cluster. One common reason is that you're trying to access the database from an IP that isn't whitelisted. Make sure your current IP address is on your Atlas cluster's IP whitelist: https://docs.atlas.mongodb.com/security-whitelist/"}
{"level":"error","message":"Error conneting to the DB: MongooseServerSelectionError: Server selection timed out after 5000 ms"}
{"level":"error","message":"Error conneting to the DB: MongooseServerSelectionError: Could not connect to any servers in your MongoDB Atlas cluster. One common reason is that you're trying to access the database from an IP that isn't whitelisted. Make sure your current IP address is on your Atlas cluster's IP whitelist: https://docs.atlas.mongodb.com/security-whitelist/"}
{"level":"error","message":"Error conneting to the DB: MongooseServerSelectionError: Could not connect to any servers in your MongoDB Atlas cluster. One common reason is that you're trying to access the database from an IP that isn't whitelisted. Make sure your current IP address is on your Atlas cluster's IP whitelist: https://docs.atlas.mongodb.com/security-whitelist/"}
{"level":"error","message":"Error conneting to the DB: MongooseServerSelectionError: Server selection timed out after 5000 ms"}
{"level":"error","message":"Error at getting new item: CastError: Cast to ObjectId failed for value \"61f7871ff398541c87003983,61fa2ecd8dd6eab6b9932d42\" (type string) at path \"_id\" for model \"carts\""}
{"level":"error","message":"Error at getting new item: CastError: Cast to ObjectId failed for value \"61f7871ff398541c87003983,61fa2ecd8dd6eab6b9932d42,61fc68bc364f4c1443ffea5b\" (type string) at path \"_id\" for model \"carts\""}
{"level":"error","message":"Error at getting new item: CastError: Cast to ObjectId failed for value \"61f7871ff398541c87003983,61fa2ecd8dd6eab6b9932d42,61fc68bc364f4c1443ffea5b,61fa2ecd8dd6eab6b9932d42\" (type string) at path \"_id\" for model \"carts\""}
{"level":"error","message":"Error at getting new item: CastError: Cast to ObjectId failed for value \"61f7871ff398541c87003983,61fa2ecd8dd6eab6b9932d42,61fc68bc364f4c1443ffea5b,61fa2ecd8dd6eab6b9932d42,61fc68bc364f4c1443ffea5b\" (type string) at path \"_id\" for model \"carts\""}
{"level":"error","message":"Error at getting user: MongooseError: Query was already executed: users.findOne({ username: 'countaleatorio@gmail.com' })"}
{"level":"error","message":"Error at getting user: MongooseError: Query was already executed: users.findOne({ username: 'countaleatorio@gmail.com' })"}
{"level":"error","message":"Error at getting user: MongooseError: Query was already executed: users.findOne({ username: 'countaleatorio@gmail.com' })"}
{"level":"error","message":"Error at getting user: MongooseError: Query was already executed: users.findOne({ username: 'countaleatorio@gmail.com' })"}
{"level":"error","message":"Error at getting user: MongooseError: Query was already executed: users.findOne({ username: 'countaleatorio@gmail.com' })"}
{"level":"error","message":"Error at getting user: MongooseError: Query was already executed: users.findOne({ username: 'countaleatorio@gmail.com' })"}
{"level":"error","message":"Error at getting user: MongooseError: Query was already executed: users.find({ username: 'countaleatorio@gmail.com' })"}
{"level":"error","message":"Error at getting user: MongooseError: Query was already executed: users.find({ username: 'countaleatorio@gmail.com' })"}
{"level":"error","message":"Error at getting user: MongooseError: Query was already executed: users.find({ username: 'countaleatorio@gmail.com' })"}
{"level":"error","message":"Error at getting user: MongooseError: Query was already executed: users.find({ username: 'countaleatorio@gmail.com' })"}
{"level":"error","message":"Error at getting user: MongooseError: Query was already executed: users.find({ username: 'countaleatorio@gmail.com' })"}
{"level":"error","message":"Error at getting user: MongooseError: Query was already executed: users.findOne({ username: 'countaleatorio@gmail.com' })"}
{"level":"error","message":"Error at getting user: MongooseError: Query was already executed: users.findOne({ username: 'countaleatorio@gmail.com' })"}
{"level":"error","message":"Error at getting user: MongooseError: Query was already executed: users.findOne({ username: 'countaleatorio@gmail.com' })"}
{"level":"error","message":"Error at getting all items: Error: ENOENT: no such file or directory, open 'C:\\Users\\tlalv\\OneDrive\\Documents\\courses\\nodejs\\desafioDiecisiete\\DB\\users.json'"}
{"level":"error","message":"Error at getting user: TypeError: Cannot read properties of undefined (reading 'find')"}
{"level":"error","message":"Error at getting user: TypeError: Cannot read properties of null (reading 'find')"}
{"level":"error","message":"Error at getting user: TypeError: users.find is not a function"}
{"level":"error","message":"Error at getting user: TypeError: users.find is not a function"}
{"level":"error","message":"Error at getting user: TypeError: users.find is not a function"}
{"level":"error","message":"Error at getting user: TypeError: users.find is not a function"}
{"level":"error","message":"Error at getting user: TypeError: Cannot read properties of null (reading 'find')"}
{"level":"error","message":"Error at getting messages: TypeError: Cannot read properties of undefined (reading 'getAll')"}
{"level":"error","message":"Error at getting messages: TypeError: Cannot read properties of undefined (reading 'getAll')"}
{"level":"error","message":"Error at getting messages: TypeError: Cannot read properties of undefined (reading 'getAll')"}
{"level":"error","message":"Error at getting messages: TypeError: Cannot read properties of undefined (reading 'getAll')"}
{"level":"error","message":"Error at getting messages: TypeError: Cannot read properties of undefined (reading 'getAll')"}
{"level":"error","message":"Error at getting new item: CastError: Cast to ObjectId failed for value \"localprice\" (type string) at path \"_id\" for model \"products\""}
{"level":"error","message":"Error at getting new item: CastError: Cast to ObjectId failed for value \"localprice\" (type string) at path \"_id\" for model \"products\""}
